symengine/cwrapper.cpp

```diff
@@ -52,6 +52,16 @@ void basic_free(basic s)
     s->m.~RCP();
 }
 
+basic_struct* basic_new_heap()
+{
+    return new CRCPBasic();
```

Comment:
Well, since this is a C++ file, they should be equivalent, and I felt it is more explicit to initialize CRCPBasic directly.